### DISCUSSION TITLE
show 'state' in windowSwitcher

### DISCUSSION THREAD

**Discussion by salahoued** [labwc/labwc#2013] [Q&A] [answered]
I started updating my `rc.xml` adding the new features and removing what I don't use/need,
In doing so I found the `state` content `windowSwitcher`, but I couldn't figure out how to properly add it.

```xml
<windowSwitcher show="yes" preview="yes" outlines="yes">
    <fields>
      <field content="type_short" width="5%"/>
      <field content="state" width="15%"/>
      <field content="identifier" width="20%"/>
      <field content="title" width="55%"/>
    </fields>
  </windowSwitcher>
```

~~But I got an empty `state` field in the `windowSwitcher`~~ (no issue here, it's just because all I had are unmax/unmin/unfull windows)

### And how can I use the `custom`content (some example)

Sorry, I couldn't get a screenshot

**Comment by Consolatis** [comment, marked as answer]
`<field content="custom" format="%W-%O" width="40%"/>`
Would for example write the current workspace and the output of the window.

**Reply by salahoued** [reply]
Thanks, I didn't see any `format` attribute in the `man page`, and didn't think about it

**Reply by Consolatis** [reply]
yep, that seems to be missing completely from the man page. It is only documented in `docs/rc.xml.all`.
